src/main/kotlin/epgx/ops/AtAtOp.kt

```diff
@@ -0,0 +1,12 @@
+package epgx.ops
+
+import org.jetbrains.exposed.sql.ComparisonOp
+import org.jetbrains.exposed.sql.Expression
+
+/**
+ * Represents an SQL operator that checks if [vector] is equals to [query].
```

Comment:
```suggestion
 * Represents an SQL operator that checks if [query] matches [vector].
```

Reply:
It works both ways so i fixed the doc accordingly